### DISCUSSION TITLE
Project 3 - plots from simplerocket / rocket

### DISCUSSION THREAD

**Discussion by C-Doll** [cooperrc/computational-mechanics#165] [Q&A]
This may just be a dumb question but I want to compare figures that I created with the simplerocket function to another figure created from the rocket function. This is to compare the Height vs Mass with and without drag. 
I'd like to overlay the plots but I used the same variables for both functions. Without changing variables to call each plot individually,  is it possible to save a plot as a variable and call back to it to plot it?

for example I tried to name my plot based on the simplerocket function as so:
simple_rocket_height_mass = plt.plot(state[:,2], state[:,0])

and then when I try to call that variable it doesn't plot anything. 

for comparison the plot from the rocket function has the exact same variables:
plt.plot(state[:,2], state[:,0])

so trying to overlay a plot with the exact same variables won't work.

is there no way to define a plot globally? I think i recall in C++ you could do that and reference it at any time.

**Comment by C-Doll** [comment]
I ended up changing each state to it's own variable. 

simple_v=state[:,2]
simple_h=state[:,0]
plt.plot(simple_v,simple_h) 

plt.plot(state[:,2], state[:,0])

and that gave me what I was looking for.

**Comment by Nadinemm5** [comment]
This is not exactly what you're looking for but using `%matplotlib notebook` or `%matplotlib widget` will have the computer default to putting all graphs into the same plot.  Every time you want something on its own plot, you can use `plt.figure()` before you plot data.
